test/models/list.rb

```diff
@@ -3,5 +3,6 @@ class List < ActiveRecord::Base
   has_many :new_items, -> { order(:position) }, dependent: :destroy
   has_many :item_without_advisory_locks, -> { order(:position) }, dependent: :destroy
   has_many :item_with_composite_primary_keys, -> { order(:position) }, dependent: :destroy
+  has_many :item_with_default_scope, -> { order(:position) }, dependent: :destroy
```

Comment:
Can we change this to `default_scope_items`? Just works better with the naming conventions. Bummer I didn't catch this for the other examples there! :D

Reply:
Makes total sense :)

Reply:
@brendon I am unsure of the model name. I did rename the models from `ItemWithDefaultScope` to `DefaultScopeItem` as well, so the `has_many` above is a bit less awkward to write (no `class_name` needed), but if I should keep it at `ItemWithDefaultScope`, let me know.

Reply:
Sorry, yes I meant change all of that also :) Thanks for that :)